airflow-core/src/airflow/ui/src/i18n/locales/he/dashboard.json

```
@@ -0,0 +1,38 @@
+{
+  "welcome": "ברוכים הבאים",
+  "stats": {
+    "stats": "סטטיסטיקות",
+    "failedDags": "בכשלון Dags",
+    "runningDags": "בריצה Dags",
+    "activeDags": "פעיל Dags",
+    "queuedDags": "Dags בתור"
+  },
+  "health": {
+    "health": "תקינות",
+    "metaDatabase": "מטא מסד-נתונים",
+    "scheduler": "מתזמן",
+    "triggerer": "Triggerer",
+    "dagProcessor": "Dag מנתח",
+    "status": "סטאטוס",
+    "lastHeartbeat": "פעימה אחרונה",
+    "healthy": "תקין",
+    "unhealthy": "לא תקין"
+  },
+  "importErrors": {
+    "searchByFile": "חפש לפי קובץ",
+    "dagImportError_one": "שגיאות בטעינת Dag",
+    "dagImportError_other": "שגיאות בטעינת Dags",
+    "timestamp": "חותמת זמן"
+  },
+  "poolSlots": "Pool סטאטוס",
+  "managePools": "Pools ניהול",
+  "history": "היסטוריה",
+  "sortBy": {
+    "newestFirst": "חדש קודם",
+    "oldestFirst": "ישן קודם"
+  },
+  "source": "מקור",
+  "group": "קבוצה",
+  "uri": "כתובת",
```

```suggestion
  "uri": "URI",
```
עדיף לדעתי להשאיר ברשאי התיבות המקוריים, כי המשמעות יותר רחבה מאשר "כתובת אינטרנט" (מזהה משאבים אחיד).